lang/py/avro/schema.py

```diff
@@ -493,6 +516,17 @@ def __eq__(self, that):
 class PrimitiveSchema(Schema):
     """Valid primitive types are in PRIMITIVE_TYPES."""
 
+    _validators = {
```

Comment:
While I started this approach, I realize now that these lambdas don't show up clearly in test coverage reports. We can leave it this way for now, but maybe in the future I should move these to named functions.